src/utils.jl

```diff
@@ -428,3 +428,33 @@ Get the amount dimension of a quantity (e.g., mol^(uamount)).
 """
 uamount(q::UnionAbstractQuantity) = uamount(dimension(q))
 uamount(d::AbstractDimensions) = d.amount
+
+"""
+Utility functions for `DynamicQuantitiesCollection`
+"""
+
+for f in (:enumerate, :length, :lastindex)
+    @eval begin
+        Base.$f(u::DynamicQuantitiesCollection) = $f(u.collection)
+    end
+end
+
+Base.getindex(u::DynamicQuantitiesCollection, i::Union{Int,  Symbol}) = getindex(u.collection, i)
+Base.getindex(u::DynamicQuantitiesCollection, i::UInt16) = getindex(u.collection, convert(Int, i))
```

Comment:
Maybe just use `Union{Integer,Symbol}` for a single method? It should automatically get converted by the underlying collection, no?

Reply:
Resolved https://github.com/SymbolicML/DynamicQuantities.jl/pull/107/commits/4ef0e83098e34666f46a866152ae183e98536168#diff-47c27891e951c8cd946b850dc2df31082624afdf57446c21cb6992f5f4b74aa2R442